README.es-ES.md

```
@@ -0,0 +1,72 @@
+
+
+# PDF Chatbot
+
+El Chatbot de PDF es una aplicación basada en la web que permite a los usuarios cargar archivos PDF y hacer preguntas sobre el contenido de dichos archivos. El chatbot utiliza OpenAI GPT y Pinecone para proporcionar respuestas precisas y relevantes basadas en los PDF cargados.
+
+## Tabla de contenido
+
+- [Instalación](#installation)
```

markdownlint-cli2 (0.23.1)

[warning] Link fragments should be valid

(MD051, link-fragments)